analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarSematicModelReportingContextTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2022 SonarSource SA
```

Comment:
```suggestion
 * Copyright (C) 2015-2023 SonarSource SA
```

Reply:
CI fails if  I do so. I will need to do a rebase at one point anyway and it should be fixed then.

Reply:
So rebase :)